src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="block-output-branching-config">
     <div class="form-group">
-      <label class="d-block">{{'flow-builder.output-branching' | trans}}</label>
+      <label :class="labelClass">{{'flow-builder.output-branching' | trans}}</label>
```

Comment:
I fixed this too!  But I liked your approach of customizing the style if needed 👍

Reply:
Sorry, I didn't see this comment before I reviewed yours :D 
Thanks